plugins/inputs/chrony/chrony.go

```diff
@@ -38,6 +41,27 @@ func (*Chrony) SampleConfig() string {
 	return sampleConfig
 }
 
+// dialUnix opens an unixgram connection with chrony
+func dialUnix(address string) (*net.UnixConn, error) {
+	dir := path.Dir(address)
+	local := path.Join(dir, fmt.Sprintf("chrony-telegraf-%s.sock", uuid.New().String()))
+	// XXX TODO: remove the file at the end
```

Comment:
TBH I don't really know how to do that cleanly. Because a defer removes it early. Could you recomend something to me ? i.e . a new `conn` struct or similar ? Thanks!